content/en/docs/reference/using-api/deprecation-guide.md

```diff
@@ -84,6 +93,16 @@ RuntimeClass in the **node.k8s.io/v1beta1** API version will no longer be served
 * All existing persisted objects are accessible via the new API
 * No notable changes
 
+### v1.24
+
+The **v1.24** release will stop serving the following deprecated API versions:
+
+#### HorizontalPodAutoscaler {#horizontalpodautoscaler-v124}
+
+The **autoscaling/v2beta1** API version of HorizontalPodAutoscaler will no longer be served in v1.24.
+
+* Migrate manifests and API clients to use the **autoscaling/v2** API version, available since v1.23.
```

Comment:
there are schema changes between v2beta1 and v2 it would be good to enumerate here. @josephburnett, can you add that content?